roles/cinder/templates/cinder.conf.j2

```diff
@@ -1,20 +1,19 @@
 # {{ansible_managed}}
 
+{% set lvm_backends = ['lvm_'] | product(cinder_lvm_pools | map(attribute='name')) | map('join') | list if 'lvm' in cinder_backends else [] %}
+{% set rbd_backends = ['rbd_'] | product(cinder_rbd_pools | map(attribute='pool')) | map('join') | list if 'rbd' in cinder_backends else [] %}
```

Comment:
Why do you use the pool here? Name would be more consistent.

Reply:
You're right, but we used the pool variable before, so I'm not sure if we run into issues with existing deployments.

Reply:
That makes sense, we can keep it.